modules/ROOT/pages/first-steps-after-setup.adoc

```diff
@@ -106,7 +106,7 @@ NOTE: do not create a user or an administrator with the same login and password
 . Click on the "More" button (in the top right corner).
```

Comment:
on the "More" button => on "More" button

Reply:
I'll rewrite this section since it's clicking on triple dots instead of "More" button. I would leave "the", since it's a specific button.